### PR TITLE
Add benefits of authenticating using Docker Desktop

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Added the recommendation on authenticating using Docker Desktop after a successful install. 
Highlight the benefits to users and admins

### CODE REVIEW

File: desktop/index.md [hunk 1]
```diff
@@ -49,6 +49,19 @@ Docker Desktop is available for Mac and Windows. For download information, syste
 
 For information about Docker Desktop licensing, see [Docker Desktop License Agreement](../subscription/index.md#docker-desktop-license-agreement).
 
+## Sign in to Docker Desktop
+
+After you’ve successfully installed and started Docker Desktop, we recommend that you authenticate using the **Sign in/Create ID** option from the Docker menu.
+
+Authenticated users get a higher pull rate limit compared to anonymous users. For example, if you are authenticated, you get 200 pulls per 6 hour period per IP address, compared to 100 pulls per 6 hour period per IP address for anonymous users. For more information, see [Download rate limit](../docker-hub/download-rate-limit.md).
+
+Authenticating through Docker Desktop also allows administrators to track the number of users who are using Docker Desktop. Additionally, through the [Image Access Management](../docker-hub/image-access-management.md) feature, administrators can configure Docker Desktop and set up guardrails to allow team members to:
```
Comment (stephen-turner) [inline desktop/index.md:58]: I know I read this once before, but now I'm wondering whether the first sentence of this is really true. Do admins have a way to count the number of users using DD? Even once we have SSO, will they?

Comment (usha-mandya) [reply desktop/index.md:58]: I thought it was possible to see how many users are running Docker Desktop when users sign in to specific organization (at least through the Docker Business subscription).

Comment (stephanierifai) [reply desktop/index.md:58]: I don't think that's true currently. CC @izzychu can you help confirm?

File: desktop/index.md [hunk 2]
```diff
@@ -49,6 +49,19 @@ Docker Desktop is available for Mac and Windows. For download information, syste
 
 For information about Docker Desktop licensing, see [Docker Desktop License Agreement](../subscription/index.md#docker-desktop-license-agreement).
 
+## Sign in to Docker Desktop
+
+After you’ve successfully installed and started Docker Desktop, we recommend that you authenticate using the **Sign in/Create ID** option from the Docker menu.
+
+Authenticated users get a higher pull rate limit compared to anonymous users. For example, if you are authenticated, you get 200 pulls per 6 hour period per IP address, compared to 100 pulls per 6 hour period per IP address for anonymous users. For more information, see [Download rate limit](../docker-hub/download-rate-limit.md).
```
Comment (stephen-turner) [inline desktop/index.md:56]: If you are authenticated, you get those pulls per user, not per IP address.

**Review by stephen-turner — CHANGES_REQUESTED** [review]
One correction.

File: desktop/index.md (hunk 2 above)
Comment (usha-mandya) [reply desktop/index.md:56]: OK, in that case, does it make sense to say "f you are authenticated, you get 200 pulls per 6 hour period, compared to 100 pulls per 6 hour period per IP address for anonymous users."

Comment (stephen-turner) [reply desktop/index.md:56]: Yes

Comment (usha-mandya) [reply desktop/index.md]: Thanks. Updated

**Review by stephen-turner — APPROVED** [review]
LGTM now thanks

**Review by stephanierifai — APPROVED** [review]
LGTM!!

**Pull request merged**